gabriel-portfolio/portfolio/src/main/webapp/index.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html>
+
+
+  <head>
+    <meta charset="UTF-8">
+    <title>Gabriel's Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+
+    <body>
```

Comment:
I think the indentation is off. <body> and <head> is at the same level. And for each level, indent by 2 spaces is recommended.